lib/challenge.js

```diff
@@ -58,8 +84,15 @@ var some_pet;
 //    the values of `name`, `species`, `breed`, and `noise`. The constructor function should also give each pet a method called `makeNoise`,
 //    which behaves as described in Q3.
 
-var Pet;
+var Pet = function Pet (name,species,breed,noise) {
+  this.name = name;
+  this.species = species;
+  this. breed = breed;
+  this.noise = noise;
+
+}
 // Answer Starts Here
+var somePet = new Pet("max", "dog"."lab","woof!");
```

Comment:
You've got a period between `"dog"` and `"lab"` where there should be a comma. It's a typo, so it won't be counted, but you should know in order to be more careful in the future.

This is otherwise a good instantiation (creating a specific pet using the `Pet` constructor).

What this question wanted in addition to the constructor function was for you to place a prototype method on it that all instances of `Pet` would then have. 1/2